docs.md

```diff
@@ -2591,7 +2591,7 @@ Equivalent to `close(2)`.
 **Parameters:**
 - `path`: `string`
 - `flags`: `string` or `integer`
-- `mode`: `integer`
+- `mode`: `integer` (octal representation of `chmod(1)` mode)
```

Comment:
How about including an example like this:

```suggestion
- `mode`: `integer` (octal `chmod(1)` mode, e.g. `tonumber('644', 8)`)
```

(this is how the mode parameter is typically constructed in [the tests](https://github.com/luvit/luv/blob/7233e6dea92498a244feb51b790c1ba51e8abbff/tests/test-fs.lua#L4))

Reply:
If you want I can add it in the other places as well? Let me know.

Reply:
Yes, please do.